tabular/src/autogluon/tabular/configs/hyperparameter_configs.py

```diff
@@ -380,16 +380,26 @@
         ],
     },
     zeroshot=hyperparameter_portfolio_zeroshot_2023,
+    zeroshot_2023=hyperparameter_portfolio_zeroshot_2023,
 )
 
 # default_FTT is experimental
 hyperparameter_config_dict["default_FTT"] = {"FT_TRANSFORMER": {}}
 hyperparameter_config_dict["default_FTT"].update(hyperparameter_config_dict["default"])
 
 # extreme is experimental
-hyperparameter_config_dict["extreme"] = {"TABPFN": {"N_ensemble_configurations": 8}}
-hyperparameter_config_dict["extreme"].update(hyperparameter_config_dict["default_FTT"])
+tabpfnmix_default = {
+    "model_path_classifier": "autogluon/tabpfn-mix-1.0-classifier",
+    "n_ensembles": 1,
+    "max_epochs": 30,
+    "ag.sample_rows_val": 5000,  # Beyond 5k val rows fine-tuning becomes very slow
+    "ag.max_rows": 200000,  # Beyond 200k rows, the time taken is longer than most users would like (hours), while the model is very weak at this size
+    "ag_args": {"name_suffix": "v1"},
+}
 
+hyperparameter_config_dict["extreme_2024"] = {"TABPFNMIX": tabpfnmix_default}
+hyperparameter_config_dict["extreme_2024"].update(hyperparameter_config_dict["zeroshot_2023"])
+hyperparameter_config_dict["extreme"] = hyperparameter_config_dict["extreme_2024"]
```

Comment:
Are we intended to keep `extreme_quality` going forward in subsequent versions? Is so, it is a bit confusing between `extreme` and `best`. If not, shall we name the preset to `experimental` or something else to indicate its experimental nature and we might merge corresponding models + HPs into `best` once stabilized

Reply:
good point, renamed to experimental_quality and added docstrings describing the preset.